migrate.py

```diff
@@ -246,58 +244,63 @@ def checkout_repo(
 
 ### FILE utils
 
-def remove(path):
+def remove(path, namespace, collection):
     global REMOVE
-    REMOVE.add(path)
+    REMOVE[namespace][collection].add(path)
 
 
-def actually_remove(checkout_path, namespace, collection):
+def actually_remove(checkout_path):
     global REMOVE
 
-    # load sanity/ignore.txt, the files being removed below need to be removed from ignore.txt too
-    sanity_ignore = read_lines_from_file(os.path.join(checkout_path, 'test/sanity/ignore.txt'))
-    new_sanity_ignore = defaultdict(list)
-    for ignore in sanity_ignore:
-        values = ignore.split(' ', 1)
-        new_sanity_ignore[values[0]].append(values[1])
-
-    # actually remove files we marked for removal
-    init_files = set()
-    for path in REMOVE:
-        actual_devel_path = os.path.relpath(path, checkout_path)
-        if actual_devel_path.endswith('__init__.py'):
-            init_files.add(path)
-            continue
+    paths = [path for namespace in REMOVE for collection in REMOVE[namespace] for path in REMOVE[namespace][collection]]
```

Comment:
There's a simpler way of doing this:
```suggestion
    paths = (
        path
        for namespace, collection in REMOVE.items()
        for path in collection
    )
```

Reply:
Your code would return names of collections.

Reply:
ah, maybe. but I hope you've got the idea of the simplification

Reply:
I wrote that in the GH UI w/o checking; here's what actually works: https://github.com/mkrizek/collection_migration/pull/1